### ISSUE TITLE
Issue when i use the extension

### ISSUE THREAD

**Issue by Macbenach** [tecnocen-com/yii2-formgenerator#10]
## **What steps will reproduce the problem?**

1) I install a advanced template from http://www.yiiframework.com/doc-2.0/guide-start-installation.html#installing-via-composer
2) I put this content in a composer.json
"require": {
        "php": ">=7.0.0",
        "bower-asset/rangeslider.js": "*",
        "bower-asset/slider-revolution": "*",
        "bower-asset/jquery_appear": "*",
        "bower-asset/jquery-countTo": "*",
        "yiisoft/yii2": "2.0.12",
        "yiisoft/yii2-bootstrap": "*",
        "yiisoft/yii2-jui": "*",
        "yiisoft/yii2-authclient": "*",
        "yiisoft/yii2-swiftmailer": "*",
        "yiisoft/yii2-apidoc": "~2.0.5",
        "tecnocen/yii2-roa": "*",
        "tecnocen/yii2-formgenerator": "dev-master",
        "filsh/yii2-oauth2-server": "2.0.3",
        "tecnocen/yii2-rmdb": "*",
        "tecnocen/yii2-workflow": "dev-master",
        "tecnocen/yii2-migrate": ">=1.0.1"
    },
3) Ran composer update in a terminal
4) Run migrations and fixtures from the extension
5) Put this configuration in my main.php

'api' => [
            'class' => 'backend\\api\\VersionContainer',
        ],
        'rmdb' => [
            'class' => 'tecnocen\\rmdb\\Module',
        ],
    ],

6) Configure my api v1 for the routes for extension, here a example:

'field' => [
            'class' => \tecnocen\formgenerator\roa\resources\FieldResource::class,
        ],
        'field/<field_id:\d+>/rule' => [
            'class' => \tecnocen\formgenerator\roa\resources\field\RuleResource::class,
        ],
        'field/<field_id:\d+>/rule/<rule_id:\d+>/property' => [
            'class' => \tecnocen\formgenerator\roa\resources\field\rule\PropertyResource::class,
            'urlRule' => ['tokens' => ['{id}' => '<id:\w+>']],
        ],

In the section $resources in Version.php from tecnocen\roa

## **What is the expected result?**

Display the ruyles for CRUD from fields, form, rule, etc.

## **What do you get instead?**
```

In all routes i get the output:

{
"name": "Unknown Property",
"message": "Getting unknown property: tecnocen\\rmdb\\Module::userId",
"code": 0,
"type": "yii\\base\\UnknownPropertyException",
"file": "/var/www/html/formulario/vendor/yiisoft/yii2/base/Component.php",
"line": 147,
"stack-trace":[
"#0 /var/www/html/formulario/vendor/yiisoft/yii2/di/ServiceLocator.php(75): yii\\base\\Component->__get('userId')",
"#1 /var/www/html/formulario/vendor/tecnocen/yii2-rmdb/src/models/Pivot.php(56): yii\\di\\ServiceLocator->__get('userId')",
"#2 /var/www/html/formulario/vendor/tecnocen/yii2-rmdb/src/models/Entity.php(37): tecnocen\\rmdb\\models\\Pivot->behaviors()",
"#3 /var/www/html/formulario/vendor/tecnocen/yii2-formgenerator/src/roa/models/SlugTrait.php(20): tecnocen\\rmdb\\models\\Entity->behaviors()",
"#4 /var/www/html/formulario/vendor/yiisoft/yii2/base/Component.php(650): tecnocen\\formgenerator\\roa\\models\\Field->behaviors()",
"#5 /var/www/html/formulario/vendor/yiisoft/yii2/base/Component.php(533): yii\\base\\Component->ensureBehaviors()",
"#6 /var/www/html/formulario/vendor/yiisoft/yii2/db/BaseActiveRecord.php(898): yii\\base\\Component->trigger('init')",
"#7 /var/www/html/formulario/vendor/yiisoft/yii2/base/Object.php(107): yii\\db\\BaseActiveRecord->init()",
"#8 /var/www/html/formulario/vendor/yiisoft/yii2/data/ActiveDataProvider.php(180): yii\\base\\Object->__construct()",
"#9 /var/www/html/formulario/vendor/yiisoft/yii2/data/BaseDataProvider.php(234): yii\\data\\ActiveDataProvider->setSort(Array)",
"#10 /var/www/html/formulario/vendor/yiisoft/yii2/data/ActiveDataProvider.php(112): yii\\data\\BaseDataProvider->getSort()",
"#11 /var/www/html/formulario/vendor/yiisoft/yii2/data/BaseDataProvider.php(101): yii\\data\\ActiveDataProvider->prepareModels()",
"#12 /var/www/html/formulario/vendor/yiisoft/yii2/data/BaseDataProvider.php(114): yii\\data\\BaseDataProvider->prepare()",
"#13 /var/www/html/formulario/vendor/yiisoft/yii2/rest/Serializer.php(186): yii\\data\\BaseDataProvider->getModels()",
"#14 /var/www/html/formulario/vendor/yiisoft/yii2/rest/Serializer.php(152): yii\\rest\\Serializer->serializeDataProvider(Object(yii\\data\\ActiveDataProvider))",
"#15 /var/www/html/formulario/vendor/yiisoft/yii2/rest/Controller.php(99): yii\\rest\\Serializer->serialize(Object(yii\\data\\ActiveDataProvider))",
"#16 /var/www/html/formulario/vendor/yiisoft/yii2/rest/Controller.php(77): yii\\rest\\Controller->serializeData(Object(yii\\data\\ActiveDataProvider))",
"#17 /var/www/html/formulario/vendor/yiisoft/yii2/base/Controller.php(158): yii\\rest\\Controller->afterAction(Object(yii\\rest\\IndexAction), Object(yii\\data\\ActiveDataProvider))",
"#18 /var/www/html/formulario/vendor/yiisoft/yii2/base/Module.php(523): yii\\base\\Controller->runAction('index', Array)",
"#19 /var/www/html/formulario/vendor/yiisoft/yii2/web/Application.php(102): yii\\base\\Module->runAction('api/v1/field/in...', Array)",
"#20 /var/www/html/formulario/vendor/yiisoft/yii2/base/Application.php(380): yii\\web\\Application->handleRequest(Object(yii\\web\\Request))",
"#21 /var/www/html/formulario/backend/web/index.php(17): yii\\base\\Application->run()",
"#22 {main}"
]
}

```

**Comment by Faryshta** [comment]
what do you get when you run

`composer show tecnocen/yii2-rmdb`?

**Comment by Macbenach** [comment]
That is the output:

name     : tecnocen/yii2-rmdb
descrip. : Yii 2 Library to create RMDB models and migrations
keywords : advanced, framework, migrate, migration, relational model database, rmdb, yii2
versions : * 0.1.0
type     : library
license  : BSD 3-clause "New" or "Revised" License (BSD-3-Clause) (OSI approved) https://spdx.org/licenses/BSD-3-Clause.html#licenseText
source   : [git] https://github.com/tecnocen-com/yii2-rmdb.git 58ccf6510e0b7cf83ca42e2a7eba8950f78c5148
dist     : [zip] https://api.github.com/repos/tecnocen-com/yii2-rmdb/zipball/58ccf6510e0b7cf83ca42e2a7eba8950f78c5148 58ccf6510e0b7cf83ca42e2a7eba8950f78c5148
names    : tecnocen/yii2-rmdb

autoload
psr-4
tecnocen\rmdb\ => src/

requires
php >=5.5
tecnocen/yii2-migrate *
yiisoft/yii2 ~2.0.10

requires (dev)
codeception/base ^2.2.3
codeception/verify ~0.3.1

suggests
yii2tech/ar-softdelete Needed for Persistent Entities

**Comment by Faryshta** [comment]
please upgrade to recently released 0.1.1
